.github/workflows/remote-integ-tests-workflow.yml

```diff
@@ -34,23 +29,33 @@ jobs:
           distribution: 'corretto'
           java-version: '11'
 
+      - uses: actions/checkout@v2
+      - name: Fetch Version Information
+        run: |
+          OPENSEARCH_VERSION=$(node -p "require('./package.json').version")
+          BRANCH_NAME=${GITHUB_REF#refs/heads/}
```

Comment:
This doesn't seem to work on PRs, since the refs/heads seems to point to the PR's ref.

Reply:
right, so I modified the logic by 
if it's a pull request, get the branch name by the pull_request.base,
if it's push, get the branch name by the github.ref, 
it seems working from the log here https://github.com/opensearch-project/dashboards-search-relevance/actions/runs/4887272941/jobs/8723655835?pr=177#step:3:16